test/Microsoft.TemplateEngine.Authoring.CLI.IntegrationTests/Microsoft.TemplateEngine.Authoring.CLI.IntegrationTests.csproj

```diff
@@ -15,5 +15,12 @@
   <ItemGroup>
     <Compile Include="..\Shared\**\*.cs" Link="%(RecursiveDir)%(Filename)%(Extension)" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Compile Remove="Snapshots\**\*" />
```

Comment:
Isn't it an extra part?

Reply:
This is needed to include Snapshots folder as part of the project as content.